lib/sqlalchemy/dialects/postgresql/array.py

```diff
@@ -4,15 +4,20 @@
 #
 # This module is part of SQLAlchemy and is released under
 # the MIT License: https://www.opensource.org/licenses/mit-license.php
-# mypy: ignore-errors
 
 
 from __future__ import annotations
 
 import re
-from typing import Any
+from typing import Any as Any_
+from typing import Iterable
 from typing import Optional
+from typing import Self
```

Comment:
iirc we need to import this from util.typing since it's not present in all python version we support